app/src/main/java/com/kickstarter/models/Update.java

```diff
@@ -18,6 +18,7 @@
   public abstract @Nullable Integer commentsCount();
   public abstract @Nullable Boolean hasLiked();
   public abstract long id();
+  public abstract boolean isPublic();
```

Comment:
ran into an unexpected wall--looks like there's no straightforward way of deserializing this reserved name (`"public"`) while using `@AutoParcel` so i may just leave this field out until we actually need it so I can move on 😬

discussion: https://github.com/frankiesardo/auto-parcel/issues/6